project/README.rst

```diff
@@ -98,4 +98,4 @@ Project List by Account Name
    `RIL <README.rst>`__: https://github.com/ottersome/RIL
 
 #. `BlenderWang9487 <https://github.com/BlenderWang9487>`__ for
-   `DummyML <BlenderWang9487/README.rst>`__: https://github.com/BlenderWang9487/DummyML
\ No newline at end of file
+   `DummyML <BlenderWang9487/README.rst>`__: https://github.com/BlenderWang9487/DummyML
```

Comment:
Please add an entry pointing to your `simpson0114/README.rst`.

Reply:
OK, I have added an entry pointing to my github readme.